src/routes/namespaces/[namespace]/workflows/_workflows-summary-row.svelte

```diff
@@ -33,18 +33,18 @@
   </td>
   <td>
     <a sveltekit:noscroll {href}>
-      <div class={`flex justify-center  `}>
+      <div class={`flex justify-start`}>
         <WorkflowStatus status={workflow.status} />
       </div>
     </a>
   </td>
   <td>
-    <a sveltekit:noscroll {href} class="font-mono text-right">
+    <a sveltekit:noscroll {href} class="text-left">
       <Time time={workflow.startTime} {timeFormat} />
     </a>
   </td>
   <td>
-    <a sveltekit:noscroll {href} class="font-mono text-right">
+    <a sveltekit:noscroll {href} class="text-let">
```

Comment:
Should this be `text-left`?

Reply:
Oops yes let me push up the new changes in a few this reverts back the changes in the last commit back to a window type view